### PR TITLE
WIP: Airlock integration

### PR DESCRIPTION
**Pull request by dfangshuo**
## Integrate Airlock

Integrate Airlock

### Related PRs

N/A

### Migrations

N/A

### Tests Performed, Edge Cases

N/A

### Screenshots

CC: @aivantg

### CODE REVIEW

File: src/lib/auth.js [hunk 1]
```diff
@@ -1,64 +1,48 @@
...
   });
 };
 
+const loginUser = async (email, passwordHash) => {
+  try {
+    const { user, token } = await base.login({
+      username: email,
+      password: passwordHash
+    });
+    console.log('SEE ME');
```
Comment (aivantg) [inline src/lib/auth.js]: can we remove these? haha

File: src/secret.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const clientId =
+  'AQmImilHhbyAhrxRKFQFLw0NLiyhR10m3yR4snUGC1nQQzLLggGZg6fUfVhiNHi8IgC9stRm4NBVulrd';
```
Comment (aivantg) [inline src/secret.js]: which....key is this? Only airlock key is allowed to be in git I think?

File: src/lib/auth.js [hunk 3]
```diff
@@ -1,64 +1,48 @@
...
+    console.log(user);
+    console.log(token);
```
Comment (aivantg) [inline src/lib/auth.js]: shouldn't this token be saved into the cookie?

Comment (arpanpal) [reply src/lib/auth.js]: The user and token are already stored in localStorage if you need to access in the browser. However, you should always be able to pull from base.getUser() or base.user to get the user, so don't worry about localStorage/cookies. Just know that base.getUser() works!

File: src/lib/auth.js [hunk 4]
```diff
@@ -1,64 +1,48 @@
...
 const cookies = new Cookies();
 
 const BASE_ID = 'appFaOwKhMXrRIQIp';
 
 const Airtable = require('airtable');
 
-const base = new Airtable({ apiKey: key }).base(BASE_ID);
+Airtable.configure({
+  endpointUrl: 'https://airlock-ppsolar.calblueprint.org/',
+  apiKey: 'airlock'
```
[the remark this answers is not in the document]
Comment (ethanlee16) [reply src/lib/auth.js]: Yes, this is safe! We do this to continue causing the Airtable JS client to throw when an API key isn't specified (which is the expected behavior before we forked). More info here: https://www.notion.so/calblueprint/User-Authentication-and-Token-Management-72067d56addf4959a152bfbc53bca089
